app/webpacker/components/ContactsPage/ContactForm.jsx

```diff
@@ -42,12 +42,12 @@ export default function ContactForm({ loggedInUserData }) {
     switch (selectedContactRecipient) {
       case CONTACT_RECIPIENTS_MAP.competition:
         return Competition;
-      case CONTACT_RECIPIENTS_MAP.communicationsTeam:
-        return CommunicationsTeam;
-      case CONTACT_RECIPIENTS_MAP.resultsTeam:
-        return ResultsTeam;
-      case CONTACT_RECIPIENTS_MAP.softwareTeam:
-        return SoftwareTeam;
+      case CONTACT_RECIPIENTS_MAP.wct:
+        return Wct;
+      case CONTACT_RECIPIENTS_MAP.wrt:
+        return Wrt;
+      case CONTACT_RECIPIENTS_MAP.wst:
+        return Wst;
```

Comment:
The components themselves can probably still be called `SoftwareTeam` etc. (Or maybe even `SoftwareTeamContact`?)
I mean, it literally doesn't matter what the functional React component is called.

Reply:
Yes I agree. But I named as the friendly ID to avoid too many types of namings. I'm fine to change to `SoftwareTeamContact` or `WstContact` or whatever you prefers to if needed.